source/_posts/2016-02-15-multi-room-audio-with-snapcast.markdown

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title: "Multi-room audio with Snapcast, Mopidy, and Home Assistant"
+date: 2016-02-15 17:10:56 -0500
```

Comment:
I used `rake new_post["title"]` and didn't get a `date_formatted` key ... I'll add it though.